testsuite/features/init_clients/sle_minion.feature

```diff
@@ -60,7 +60,7 @@ Feature: Bootstrap a Salt minion via the GUI
     And I wait at most 600 seconds until event "Product Migration" is completed
     And I wait until event "Package List Refresh" is completed
     And I follow "Details" in the content area
-    Then I wait until I see "SUSE Linux Enterprise Server 15 SP3" text, refreshing the page
+    Then I wait until I see "SUSE Linux Enterprise Server 15 SP4" text, refreshing the page
```

Comment:
`(BETA)` or not ?

Reply:
In the testsuite and my local tests this did not fail, so no `(BETA)`.

Reply:
It will not fail in any case, as we are looking for this text only.
So both "SUSE Linux Enterprise Server 15 SP4" and "SUSE Linux Enterprise Server 15 SP4 my funny words after my text" will match. Which is fine in that scenario.

Reply:
OK, that was my assumption, but I wanted to make sure it was intentional.